Formula/luau.rb

```diff
@@ -0,0 +1,22 @@
+class Luau < Formula
+  desc "Fast, safe, gradually typed embeddable scripting language derived from Lua"
+  homepage "https://luau-lang.org"
+  url "https://github.com/Roblox/luau/archive/0.504.tar.gz"
+  sha256 "544b3107e1c53f8349b1e327efed55ac29cc20a2983dd23031ab07c5b6944633"
+  license "MIT"
+  revision 0
+  head "https://github.com/Roblox/luau.git", branch: "master"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+    system "cmake", "--build", ".", "--target", "Luau.Repl.CLI", "--config", "RelWithDebInfo"
+    system "cmake", "--build", ".", "--target", "Luau.Analyze.CLI", "--config", "RelWithDebInfo"
```

Comment:
We don't want to have debug info for everyone
```suggestion
    system "cmake", "--build", ".", "--target", "Luau.Repl.CLI"
    system "cmake", "--build", ".", "--target", "Luau.Analyze.CLI"
```

Reply:
Why don't we want to use the default targets?

Reply:
If I don't specify the targets, it also builds the tests which is unnecessary.

Reply:
You can disable the tests with `LUAU_BUILD_TESTS=OFF`.